oe_content.module

```diff
@@ -23,11 +23,12 @@ function oe_content_entity_base_field_info(EntityTypeInterface $entity_type) {
   }
 
   $fields['oe_content_short_title'] = BaseFieldDefinition::create('string')
-    ->setLabel(t('Short title'))
+    ->setLabel(t('Alternative title'))
+    ->setDescription(t('Use this field to create an alternative title for use in the URL and in list views. If the page title is longer than 60 characters, you can add a shorter title here.'))
     ->setRequired(FALSE)
     ->setTranslatable(TRUE)
     ->setRevisionable(TRUE)
-    ->setSetting('max_length', 255)
+    ->setSetting('max_length', 170)
```

Comment:
Shouldn't we handle this like we handle the title max length? With the maxlength module? Like we limit [for the title field](https://github.com/openeuropa/oe_content/blob/master/modules/oe_content_page/config/install/core.entity_form_display.node.oe_page.default.yml#L143). What you did here only controls the max size that fits in the DB. And in any case, we should not change that from 255 because it will become impossible to change back if we ever need to. We don't change any other right?